frontend/src/app/actions/experiments.js

```
@@ -4,22 +4,13 @@ export default createActions({
 
   updateExperiment: (addonID, data) => ({ addonID, data }),
 
-  fetchExperiments: (experimentsUrl, countsUrl) => {
-    let experiments = [];
-    return fetch(experimentsUrl)
-      .then(response => response.json())
-      .then(data => {
-        experiments = data.results;
-        return fetch(countsUrl);
-      })
-      .then(response => response.json())
+  fetchUserCounts: (countsUrl) => {
+    return fetch(countsUrl)
+      .then(
+        response => response.json(),
+        () => ({}))
```

meh on errors

---

:| I'm not. This is a really good way to suppress exceptions. Can we add some error handling here?

---

I dunno, error handling here feels excessive. If an error happens should we just not have the counts (like this) or fall back to something else (which can get complicated)? If we reported an error somewhere, who's going to look at it and be able to act on it? Also there's zero error handling in the current code either (not an excuse, but we're not having problems with it either)

That's why I went with 'meh'. If you have ideas for handling it I'm all ears.

---

Good discussion here. My thoughts:

> If an error happens should we just not have the counts (like this) or fall back to something else (which can get complicated)?

I agree that we should fail gracefully, and continue rendering the page without the counts.

> If we reported an error somewhere, who's going to look at it and be able to act on it? 

If the counts can't be loaded, it might just be a local network error, or it might indicate a busted release. How about [forwarding the error to sentry](https://docs.sentry.io/clients/javascript/integrations/react/#expanded-usage), so Ops can determine if it's a global or local thing?

While you're at it, would you mind also adding a sentry ping in the [`subscribeToBasket` error handler](https://github.com/mozilla/testpilot/blob/master/frontend/src/app/containers/App.js#L123)? (Since this is a really nice way to detect problems with other services, I'll probably change #2231 to straight-up replace the `sendBeacon` ping centre call with a `fetch` that forwards network errors to Sentry.)

(Optional) If you're up for getting a little fancy, since I'm suggesting adding a `Raven.captureException` call in two spots, you could centralize that into some kind of app-global network error logging function, or DRY things up by wrapping the `fetch` API in a function that forwards exceptions to Sentry automatically...whatever, your call.

---

Logging is fine but should be worked on as a separate issue (#2253). I'm not going to do it in this PR.